chain/testing.go

```diff
@@ -367,10 +367,12 @@ func (f *Builder) GetTipSet(key types.TipSetKey) (types.TipSet, error) {
 	return types.NewTipSet(blocks...)
 }
 
-// FetchTipSets returns `recur` tipsets from `key` by following parent keys.
-func (f *Builder) FetchTipSets(ctx context.Context, key types.TipSetKey, recur int) ([]types.TipSet, error) {
+// FetchTipSets fetchs the tipset at `tsKey` from the network using the fetchers blockStore backed by the Builder.
```

Comment:
You should scare quote "network" or somehow otherwise indicate that this doesn't actually go to the network.